infra/base-images/base-builder/install_go_CodeIntelligenceTesting.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -eux
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Install CodeIntelligenceTesting Go
+#  (require at least Go1.4 to boostrap)
+cd "/root"
+git clone https://github.com/CodeIntelligenceTesting/go.git .go-CodeIntelligenceTesting
+cd ".go-CodeIntelligenceTesting/src"
+git checkout dev.libfuzzer.18
+# Disable tests, which fails
+sed -i '/^exec .* tool dist test -rebuild "$@"/ s/./#&/' run.bash
+./all.bash
+
+# Remember to set env in dockerfile before this line
+echo 'Set "GOCODEINTELLIGENCETESTINGPATH=/root/go-CodeIntelligenceTesting"'
+echo 'Set "PATH="/root/.go-CodeIntelligenceTesting/bin:$GOCODEINTELLIGENCETESTINGPATH/bin:$PATH"'
+
+# Get go114-fuzz-build with the new Go
+rm "$GOPATH/bin/go-fuzz"
```

Comment:
Can we just clear everything in `$GOPATH`? i.e. 

```bash
rm -rf $GOPATH
mkdir -p $GOPATH
```

Reply:
Sure, done in [this commit](https://github.com/google/oss-fuzz/commit/f1558c465a329203283080392f21c27d74a7116f).